src/snoowrap.d.ts

```diff
@@ -158,6 +158,7 @@ declare namespace Snoowrap {
     clientId: string;
     clientSecret?: string;
     redirectUri: string;
+    permanent?: boolean; // defaults to true
```

Comment:
I don't like giving permanent an option here as it's not used in `fromAuthCode`. Rather, add `permanent` directly to the type annotation for `fromApplicationOnlyAuth`.

Reply:
Thanks for the review!

I think that would also affect the already existing `deviceId` parameter, should we split the `fromAuthCode` and `fromApplicationOnlyAuth` options annotation?